ui/src/alerting/components/endpoints/EndpointTypeDropdown.tsx

```diff
@@ -0,0 +1,71 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {Dropdown} from '@influxdata/clockface'
+
+// Types
+import {NotificationEndpointType} from 'src/types'
+
+interface EndpointType {
+  id: NotificationEndpointType
+  type: NotificationEndpointType
+  name: string
+}
+
+interface Props {
+  selectedType: string
+  onSelectType: (type: NotificationEndpointType) => void
+}
+
+const types: EndpointType[] = [
+  {name: 'Slack', type: 'slack', id: 'slack'},
+  {name: 'Pagerduty', type: 'pagerduty', id: 'pagerduty'},
```

Comment:
```suggestion
  {name: 'PagerDuty', type: 'pagerduty', id: 'pagerduty'},
```
where's your respect for The Brand???